ambry-api/src/main/java/com/github/ambry/account/AccountService.java

```diff
@@ -107,6 +108,36 @@
    */
   public boolean removeAccountUpdateConsumer(Consumer<Collection<Account>> accountUpdateConsumer);
 
+  /**
+   * Add a new container to an existing account.
+   * @param accountName the name of account in which new container will be added.
+   * @param newContainer the new container to add.
+   * @return a pair of account id and container id strings associated with existing account and new container.
+   */
+  default Pair<String, String> addContainer(String accountName, Container newContainer) {
```

Comment:
The PostAccountsHandler will have method updateContainers that takes a collection of (new or updated) containers in the payload.  Would it make more sense to mirror that in this interface?  Otherwise the handler will need to get the existing containers, compare with the input and call add() and update() as appropriate.  Which may be okay.  But the MySqlAccountService should ideally make all the updates in a single transaction (in case multiple containers are added/updated).